plugins/python/python_plugin.c

```diff
@@ -1092,7 +1092,6 @@ void *uwsgi_python_create_env_holy(struct wsgi_request *wsgi_req, struct uwsgi_a
 }
 
 void uwsgi_python_destroy_env_holy(struct wsgi_request *wsgi_req) {
-	Py_DECREF((PyObject *)wsgi_req->async_environ);
```

Comment:
yup, wasn't sure about this.  but there seem to be two `DECREF`s straight after each other

I don't know the code well enough to know where ownership belongs.  maybe if I put a small test together than can be run under different threading/async modes?